app/controllers/orgs_controller.rb

```diff
@@ -113,8 +114,8 @@ def shibboleth_ds
   # POST /orgs/shibboleth_ds
   # rubocop:disable Metrics/AbcSize
   def shibboleth_ds_passthru
-    if !shib_params["shib-ds"][:org_name].blank?
-      session["org_id"] = shib_params["shib-ds"][:org_name]
+    if !shib_params["shib-ds"][:org_id].blank?
+      session["org_id"] = shib_params["shib-ds"][:org_id]
```

Comment:
Yes, storing the org_id in the session variable called 'org_id' makes sense. ha ha. good catch